funnel/templates/project.html.jinja2

```diff
@@ -15,7 +15,7 @@
       "@type": "Event",
       "name": {{ project.title|tojson }},
       "url": {{ project.url_for(_external=true)|tojson }},
-      "startDate": {{ project.schedule_start_at.astimezone(project.timezone).isoformat()|tojson }},
+      "startDate": {{ project.schedule_start_at_localized.isoformat()|tojson }},
```

Comment:
This will break if date is `None`. We need either (a) `tojson` to automatically render to `isoformat()` or (b) move the isoformat call into a model property that checks for `None`. Solution (a) is preferred and can be achieved by checking for a datetime type here: https://github.com/hasgeek/baseframe/blob/master/baseframe/__init__.py#L93

Reply:
Change required in Baseframe:

1. Add an import to `__init__.py`:

    ```python
    from datetime import date, datetime, time
    ```

2. Add an `if` condition in `JSONEncoder.default` following the `BaseTzInfo` condition (on line 98):

    ```python
    if isinstance(o, (date, datetime, time)):
        return o.isoformat()
    ```

Reply:
https://github.com/hasgeek/baseframe/pull/272